interop/stress/client/main.go

```diff
@@ -76,7 +87,8 @@ func parseTestCases(testCaseString string) []testCaseWithWeight {
 			panic(fmt.Sprintf("invalid test case with weight: %s", str))
 		}
 		// Check if test case is supported.
-		switch testCase[0] {
+    testCaseName := strings.ToLower(testCase[0])
+		switch strings.ToLower(testCaseName) {
```

Comment:
Repeated `ToLower`.

Consider renaming `testCase` to something else like `testCaseNameAndWeight` since this new variable name plus the old one were confusing to me.

Reply:
Renamed the variable and removed the extra call to `ToLower`